pyrefly/lib/alt/class/pydantic.rs

```diff
@@ -53,6 +57,45 @@ use crate::error::context::ErrorInfo;
 use crate::types::class::Class;
 use crate::types::types::Type;
 
+fn int_literal_from_type(ty: &Type) -> Option<&LitInt> {
```

Comment:
Can you merge this with https://github.com/facebook/pyrefly/blob/2276094063b6126db59e20608aa380aa451b9836/pyrefly/lib/alt/class/pydantic.rs#L417-L424?